### ISSUE TITLE
Sign name replacements

### ISSUE THREAD

**Issue by MrLogarithm** [MrLogarithm/cdli-accounting-viz#9]
For signs with multiple names, standardising the spelling can improve accuracy of commodity id: e.g. the dictionary recognizes *zid2* as "flour" but not *zi3*, although they are apparently the same sign.

**Comment by willismonroe** [comment]
Can you use https://github.com/tosaja/Nuolenna/blob/master/sign_list.txt to make a dictionary of multi-value signs.

zid₂ and zi₃ resolve to the same unicode glyph in that list.

**Comment by MrLogarithm** [comment]
Good idea, I also found http://etcsl.orinst.ox.ac.uk/edition2/signlist.php which could be used for the same purpose.

**Comment by willismonroe** [comment]
That's a good list too, but it's restricted to literary texts so might not
include some administrative values or signs.
